### PR TITLE
a must-have angular directive

### PR DESCRIPTION
**Pull request by yveoch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,8 @@ Open source sometimes is awesome.
 ### Comments
 
 - [Revolution](https://github.com/fre5h/DoctrineEnumBundle/pull/12#issuecomment-33023169)
+
+
+### Angular
```
Comment (bernardobarreto) [inline README.md:16]: I think this should be named "Projects".

**Comment by mehulkar** [comment]
Hard to argue that an entire project fits into the Github specific wall of fame. @Dryvenn what's the reasoning for adding this here?

**Comment by bernardobarreto** [comment]
I agree with @mehulkar, otherwise lolcat should at the top of list.

**Comment by yveoch** [comment]
Thing is, an angular directive is meant to be a very small functionnal part of a bigger project. This one does nothing except drawing a unicorn in your website. Thought that guy's work deserved some fame.

File: README.md (hunk 1 above)
Comment (CatTail) [inline README.md:16]: How about "repo" like #11

**Comment by mehulkar** [comment]
Added some reasoning to #11. I had no idea that I would
actually need to figure out some ground rules for this :)

File: README.md (hunk 1 above)
Comment (mehulkar) [inline README.md:16]: Added some reasoning to #11. I think it's in the same boat as this addition.

**Comment by mehulkar** [comment]
Closing in deference to the newly added contributing guidelines 857876fed17c4f99edd9b9d29e6cb8a63f4b79a8. 

Feel free to deliver a compelling argument and we can reopen!

**Pull request closed without merging**